### PR TITLE
Tile

### PR DESCRIPTION
**Pull request by samanthabaker**

### CODE REVIEW

File: src/components/03-components/06-dashboard/16-tile/tile--outline.njk [hunk 1]
```diff
@@ -1,59 +1,63 @@
...
-    </a>
-  </div>
\ No newline at end of file
+  <div class="tablet:grid-col">
+    <div class="sds-tile--outline flex-justify-center">
+      <h3>Register Your Entity</h3>
+      <p>Assistance listings are detailed public descriptions of federal programs that provide grants, loans, scholarships, insurance, and other types of assistance awards. You may browse assistance listings across all government agencies to form a “big
+        picture” of potential funding options.</p>
+      <button class="sds-button sds-button--icon desktop:width-full margin-y-2">
+        <span class="fa-stack fa-2x">
```
Comment (ascreven-gsa) [inline src/components/03-components/06-dashboard/16-tile/tile--outline.njk:55]: Please use layered icon component

File: src/components/03-components/06-dashboard/16-tile/tile--outline.njk [hunk 2]
```diff
@@ -1,59 +1,63 @@
...
-        </button>
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col-7">
+    <div class="sds-tile--outline">
+      <h3>The Official U.S. Government System for:</h3>
+      <div class="grid-row">
+        <div class="grid-col">
+          <ul class="sds-list  sds-list--unstyled sds-list-link">
+            <li>
+              <a href=" " class=" usa-link"=" usa-link""="usa-link""">Contract Opportunities</a>
```
Comment (ascreven-gsa) [inline src/components/03-components/06-dashboard/16-tile/tile--outline.njk:9]: Please update the class for links

File: src/components/03-components/06-dashboard/16-tile/tile.njk [hunk 3]
```diff
@@ -1,19 +1,38 @@
...
-<div class="sds-tile">
-  <a href="#" class="sds-tile__content">
-    {% render '@layered-icon', tiles[0].icon %}
-    <div>
-      <div class="sds-tile__title">Tile Title</div>
-      <div class="sds-tile__details">tile details</div>
+<div class="grid-row flex-align-center">
+<div class="tablet:grid-col">
+  <div class="sds-tile">
+    <span class="sds-tile__icon fa-stack fa-2x">
```
Comment (ascreven-gsa) [inline src/components/03-components/06-dashboard/16-tile/tile.njk:4]: Please add back instances of layered icon component

File: src/components/03-components/06-dashboard/16-tile/tile.njk [hunk 4]
```diff
@@ -1,19 +1,38 @@
-<h3 class="margin-top-0">Basic Tile Example</h3>
-<div class="sds-tile">
-  <a href="#" class="sds-tile__content">
-    {% render '@layered-icon', tiles[0].icon %}
-    <div>
-      <div class="sds-tile__title">Tile Title</div>
-      <div class="sds-tile__details">tile details</div>
+<div class="grid-row flex-align-center">
```
Comment (ascreven-gsa) [inline src/components/03-components/06-dashboard/16-tile/tile.njk:1]: Is there a reason the same tile is being displayed three times?

File: src/stylesheets/components/_tile.scss [hunk 5]
```diff
@@ -1,150 +1,57 @@
...
-    height: 150px;
-    width: 195px;
+//****************************
+//SDS TILE OUTLINE
+.sds-tile--outline {
+  @include u-border('2px', 'white');
+  @include u-padding(4);
+  @include u-bg('accent-cool-lighter');
+  @include u-width('full');
+  border-radius: 20px;
```
Comment (ascreven-gsa) [inline src/stylesheets/components/_tile.scss:36]: Please use border radius and opacity tokens

File: src/components/03-components/06-dashboard/16-tile/tile--outline.njk [hunk 6]
```diff
@@ -1,59 +1,63 @@
...
-      <div class="mobile-details mobile-lg:flex-wrap">
-        <button class="sds-button sds-button--icon margin-top-2">
-          {% render '@layered-icon', buttons[0].icons %}
-          <span class="sds-button--icon__title">Register Entity</span>
-        </button>
-        <button class="sds-button sds-button--icon margin-top-2">
-          {% render '@layered-icon', buttons[1].icons %}
-          <span class="sds-button--icon__title">Check Status</span>
-        </button>
+<div class="grid-row grid-gap">
```
Comment (ascreven-gsa) [inline src/components/03-components/06-dashboard/16-tile/tile--outline.njk:1]: Please separate into two different variants for eventual reuse

File: src/components/03-components/06-dashboard/16-tile/tile.njk (hunk 4 above)
Comment (samanthabaker) [reply src/components/03-components/06-dashboard/16-tile/tile.njk:1]: yep, to show how they behave in a row from desktop to mobile

File: src/stylesheets/components/_tile.scss (hunk 5 above)
Comment (samanthabaker) [reply src/stylesheets/components/_tile.scss:36]: there isn't a border radius token large enough in uswds to use here

**Pull request merged**